packaging/docker/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:slim
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN sed -i -e's/ main/ main contrib non-free/g' /etc/apt/sources.list.d/debian.sources \
+    && apt update \
+    && apt-get install -y netperf iperf3 iputils-ping wget \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install --no-cache-dir matplotlib flent 
```

Comment:
Why not install matplotlib from the package repositories?

Reply:
Typically Debian version of matplotlib is one major version behind (the current repo version is 3.6.3 and the pip version is 3.7.1).
Do you prefer the repo version? I don't have a strong opinion here in general, up to you.

Reply:
We don't use any bleeding edge matplotlib features, so the distro package should be fine...

Reply:
Ok, I explored package dependencies for python3-matplotlib, and at the end of the day, it brings python3-dev and g++ infrastructure, which increases the image size up to 1.2GB. 
I'd prefer to stay with the PyPI version.

Reply:
Okay, yeah, that seems a bit much; okay, let's just go with the pip version then...